Yc ks sk686 update redirection domain

Kazarus4 opened the pull request:
CCC quality check done
Links checked with gitpod
all translations done
dashboard with [!tabs] checked with gitpod for DE, FR, IT, PL, PT, ES and EN-ASIA version => ok
all screens ok

Need proofreading in EN and DE version before PR.

Commit:
en AND de TRANSLATION DONE

Commit:
en AND de TRANSLATION DONE 2

Commit:
en, es AND de TRANSLATION DONE 3

Commit:
it pl et pt translation todo

Commit:
translations done

Commit:
translations and links checked with gitpod done

Commit:
translations-links checked gitpod- dashboardchecks-done

OVH-Team-Guides commented on pages/web/domains/redirect_domain_name/guide.pl-pl.md:
```diff
@@ -1,170 +1,407 @@
 
-W celu dodania przekierowania, kliknij na przycisk `Dodaj przekierowanie`{.action} .
+- inna już istniejąca domena/subdomena:
+    - **Przykład**: `domain.tld
```
Formatting seems broken from that line

OVH-Team-Guides commented on pages/web/domains/redirect_domain_name/guide.pl-pl.md:
```diff
@@ -1,170 +1,407 @@
+### Przekieruj domenę w Panelu klienta
 
-W przypadku przekierowania widocznego, masz dwie możliwości konfigutracji.
+Zaloguj się do [Panelu klienta OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pl/&ovhSubsidiary=pl){.external}, przejdź do sekcji `Web Cloud`{.action}, wybierz domenę, którą chcesz przekierować do `sekcji Domeny`{.action}, następnie kliknij zakładkę `Przekierowanie`{.action}.
```
```suggestion
Zaloguj się do [Panelu klienta OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pl/&ovhSubsidiary=pl){.external}, przejdź do sekcji `Web Cloud`{.action}, wybierz domenę, którą chcesz przekierować do sekcji `Domeny`{.action}, następnie kliknij zakładkę `Przekierowanie`{.action}.

```

OVH-Team-Guides commented on pages/web/domains/redirect_domain_name/guide.pt-pt.md:
```diff
@@ -1,173 +1,403 @@
+A tabela apresenta os reencaminhamentos ativos para o seu domínio. Pode gerir os seus reencaminhamentos existentes utilizando o botão `...`{.action} à direita de cada linha.
 
-Uma vez escolhido o tipo de reencaminhamento visível, deverá indicar o destino do reencaminhamento (ou seja, o endereço web para o qual apontará o reencaminhamento).
+Clique no botão ``Para Adicionar um reencaminhamento`{.action}.
```
```suggestion
Clique no botão `Para Adicionar um reencaminhamento`{.action}.
```

OVH-Team-Guides commented on pages/web/domains/redirect_domain_name/guide.pt-pt.md:
```diff
@@ -1,173 +1,403 @@
+
+Esta opção permite, após a introdução do domínio reencaminhado, apresentar o domínio alvo na barra de endereços do seu browser em vez do domínio reencaminhado.
+
+- **Exemplo**: se reencaminhar `domain1.tld` para `domain2.tld`, é, é `domain2.tld` que se encontra na barra de endereços do seu browser.
```
Please check translation again, it looks weird.

Kazarus4 committed:
Update pages/web/domains/redirect_domain_name/guide.pl-pl.md

Co-authored-by: OVH-Team-Guides <43037584+OVH-Team-Guides@users.noreply.github.com>

Kazarus4 committed:
Update pages/web/domains/redirect_domain_name/guide.pt-pt.md

Co-authored-by: OVH-Team-Guides <43037584+OVH-Team-Guides@users.noreply.github.com>

Kazarus4 committed:
Update guide.pl-pl.md

Kazarus4 committed:
Update guide.pl-pl.md

Kazarus4 committed:
Translation PT

se reencaminhar `domain1.tld` para `domain2.tld`, é que está a apontar para a barra de endereços no seu browser.

bs-ovh committed:
DE proof

bs-ovh committed:
EN proof

Commit:
proof boris DE and EN update done

Y0Coss committed:
Final proofreading

Y0Coss committed:
Updating last update date
